tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -1122,6 +1122,13 @@ def defineAndResources(self, kernel, tPA, tPB, lralwaCode):
       load = self.states.numSgprToLoad
       sgprStart = self.sgprs["SizesFree"]
       moduleLoadAllKernArg = self.argLoader.loadAllKernArg(sgprStart, "KernArgAddress", load, 0)
+      if kernel["ProblemType"]["UseScaleAB"]:
+        sgprOffset = self.argLoader.getOffset()
+        for name in ['A','B']:
```

Comment:
It seems that   self.states.preloadScaleA  and   self.states.preloadScaleB can replace this kind of checking.

Reply:
Fixed